atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraConstants.java

```diff
@@ -87,7 +90,15 @@
     public static final String GLOBAL_DDL_LOCK = "Global DDL lock";
     public static final String GLOBAL_DDL_LOCK_COLUMN_NAME = "id_with_lock";
     public static final long TIME_BETWEEN_LOCK_ATTEMPT_ROUNDS_MILLIS = 1000;
-    public static final long GLOBAL_DDL_LOCK_CLEARED_VALUE = Long.MAX_VALUE;
+    static final String GLOBAL_DDL_LOCK_FORMAT = "%1$d_%2$d";
+    static final long GLOBAL_DDL_LOCK_CLEARED_ID = Long.MAX_VALUE;
+    static final String GLOBAL_DDL_LOCK_CLEARED_VALUE = String.format(
+            GLOBAL_DDL_LOCK_FORMAT, GLOBAL_DDL_LOCK_CLEARED_ID, 0);
```

Comment:
Might be worth extracting the String.format to a separate method that takes lock ID and heartbeatCount?